src/utils/Delaunay.hpp

```diff
@@ -3,22 +3,21 @@
 #include <array>
 #include <vector>
 
-namespace wmtk
-{
-	using Point3D = std::array<double, 3>;
-	using Tetrahedron = std::array<size_t, 4>;
+namespace wmtk {
+using Point3D = std::array<double, 3>;
```

Comment:
would this be potentially conflicting with others? 
Maybe we should do type alias in a common header instead of `utils`?